roxctl/generate/README.md

```diff
@@ -0,0 +1,115 @@
+# Static Network Policy Generator
+
+## Developer Preview Notice
+
+The static network policy generation feature is offered as a _developer preview_ feature.
+While we are open to receiving feedback about this feature, our technical support will be able to
```

Comment:
```suggestion
While we are open to receiving feedback about this feature, our technical support will not be able to
```

Reply:
There is no support for DP, right?

Reply:
See [edfb373](https://github.com/stackrox/stackrox/pull/2637/commits/edfb373fe06d368440609f02b30e24003870cf9a)